### PR TITLE
Add default script

### PR DESCRIPTION
**Pull request by david-buderus**
Moved default data to an external script

### CODE REVIEW

File: add_defaults.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                print(filename.removesuffix(".json") + " got created")
+
+
+
+answer = requests.get("https://uc4.cs.uni-paderborn.de/api/experimental/authentication-management/login/machine", auth=(sys.argv[1], sys.argv[2]), timeout=60)
+token = answer.json()["login"]
+
+post("https://uc4.cs.uni-paderborn.de/api/experimental/user-management/users", "user", token)
+post("https://uc4.cs.uni-paderborn.de/api/experimental/examreg-management/examination-regulations", "examreg", token)
+post("https://uc4.cs.uni-paderborn.de/api/experimental/course-management/courses", "course", token)
```
Comment (bastihav) [inline add_defaults.py]: only experimental?

**Review by bastihav — CHANGES_REQUESTED** [review]

**Review by dhelmert — APPROVED** [review]

File: defaults/user/student.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      "isActive": true,
+      "role": "Student",
+      "address": {
+          "street": "Gänseweg",
+          "houseNumber": "42a",
+          "zipCode": "13337",
+          "city": "Entenhausen",
+          "country": "Germany"
+      },
+      "firstName": "Stu",
```
Comment (this-kramer) [inline defaults/user/student.json]: I would prefer actual names for marketing reasons, the tag student should be sufficient for identifying students.

File: defaults/user/lecturer.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      "isActive": true,
+      "role": "Lecturer",
+      "address": {
+          "street": "Gänseweg",
+          "houseNumber": "42a",
+          "zipCode": "13337",
+          "city": "Entenhausen",
+          "country": "Germany"
+      },
+      "firstName": "Lect",
```
Comment (this-kramer) [inline defaults/user/lecturer.json]: See student name

**Review by this-kramer — APPROVED** [review]
Good idea. I would like to see more default data for showcasing, i.e. more students, exam regs, ....

File: defaults/user/lecturer.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      "address": {
+          "street": "Gänseweg",
+          "houseNumber": "42a",
+          "zipCode": "13337",
+          "city": "Entenhausen",
+          "country": "Germany"
+      },
+      "firstName": "Jane",
+      "lastName": "Doe",
+      "email": "jane.doet@mail.de",
```
Comment (bastihav) [inline defaults/user/lecturer.json:22]:
```suggestion
      "email": "jane.doe@mail.de",
```

**Review by bastihav — APPROVED** [review]

**Pull request merged**